File: 백준/Bronze/30802. 웰컴 키트/웰컴 키트.py
N = int(input())
sizes = list(map(int, input().split()))
T, P = map(int, input().split())

total_t_bundles = 0
for s in sizes:
    if s == 0:
        continue

    total_t_bundles += (s - 1) // T + 1

p_bundles = N // P
p_singles = N % P

print(total_t_bundles)
print(p_bundles, p_singles)

File: 백준/Bronze/30802. 웰컴 키트/웰컴 키트.js
const input = require('fs').readFileSync(0).toString().trim().split('\n');
const N = Number(input[0]);
const sizes = input[1].split(' ').map(Number);
const [T, P] = input[2].split(' ').map(Number);

let totalTshirtsBundles = 0;
for (let s of sizes) {
    totalTshirtsBundles += Math.floor((s + T - 1) / T);
}

console.log(totalTshirtsBundles);
console.log(Math.floor(N / P) + " " + (N % P));